policy/modules/system/authlogin.te

```diff
@@ -433,6 +433,11 @@ tunable_policy(`authlogin_nsswitch_use_ldap',`
 	sysnet_use_ldap(nsswitch_domain)
 ')
 
+ifdef(`init_systemd',`
+	# nss-systemd calls GetDynamicUsers() exposed by systemd over D-Bus
+	init_dbus_chat(nsswitch_domain)
+')
+
```

Comment:
This block should go above the tunable_policy authlogin_nsswitch_use_ldap block. Also waiting pending other discussions about this change on list.